tests/echo_test.rs

```diff
@@ -0,0 +1,86 @@
+/// Tests unary, server, and bidi streaming with simple echo requests. To run, simply
+/// update the credentials and uri as necessary.
+use anyhow::Result;
+use std::env;
+use viam_rust_utils::gen::proto::rpc::examples::echo::v1::echo_service_client::EchoServiceClient;
+use viam_rust_utils::gen::proto::rpc::examples::echo::v1::{
+    EchoBiDiRequest, EchoMultipleRequest, EchoRequest,
+};
+use viam_rust_utils::rpc::dial;
+
+async fn dial() -> Result<dial::ViamChannel, anyhow::Error> {
```

Comment:
(nit) you don't need to specify `anyhow::Error`, `Result<T>` is fine!

Reply:
but writing `anyhow` is just so much fun!

jk - thank you, i will update :)

Reply:
oh but doesn't this just mean i will need to specify the Result type when i call this function?